app/models/user.rb

```diff
@@ -232,10 +232,6 @@ def groups_include?(group)
     miq_group_ids.include?(group.id)
   end
 
-  def admin?
-    userid == "admin"
```

Comment:
There are six places that do that. Sure, I can leave this one then (will change the UI separately).